history/warren-weaver.md

```diff
@@ -0,0 +1,11 @@
+---
+description: One of the pioneers of machine translation
+---
+# Warren Weaver
+Warren Weaver was an American scientist who designed several methods for developing Machine translation in 1949.
```

Comment:
Let's leave out the nationality

*Machine* should be lower case

"several" can be removed?

What's the significance of 1949?  "starting in 1949" ?